include/zephyr/arch/x86/efi.h

```diff
@@ -12,10 +12,20 @@
 
 #ifndef _ASMLANGUAGE
 
+#include <stdint.h>
+
 struct efi_boot_arg {
 	void *efi_systab;           /* EFI system table */
 	unsigned long long efi_cr3; /* EFI page table */
 	void *acpi_rsdp;
+
+	/*
+	 * bootargs fields can't be placed here conditionally based on CONFIG_BOOTARGS
+	 * because this struct is also used in arch/x86/zefi/zefi.c where config defines
+	 * are unavailable
```

Comment:
I thought we had autoconf.h included in zefi.c already, no?  If not, there's no reason that can't be made to work.  Seems silly to go through the trouble of writing this comment instead of fixing the problem.